src/utilities.js

```diff
@@ -107,7 +107,10 @@ export function getRequiredCombatSkill(skillConstant) {
             skill = "magic";
             break;
         default:
-            console.log("Could not find the required combat skill.");
+            skill = "attack";
+            console.log(
+                "Could not find the required combat skill. Defaulting to unarmed."
+            );
```

Comment:
Currently prints out every time you click while unarmed. Can be fixed by adding an optional parameter
```js
export function getRequiredCombatSkill(skillConstant, allowUnarmed = false) {
  //...
  default:
    if (!unarmed) {
      console.log('Could not find the required combat skill. Defaulting to attack.');
    }
    return attack
}
```